Source/Public/Publish-Bicep.ps1

```diff
@@ -0,0 +1,46 @@
+function Publish-Bicep {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 1)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path,
+
+        [Parameter(Mandatory, Position = 2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Target          
```

Comment:
we are dependent on the format of the target being correct and should validate it to be, either by using a validate script,
or implementing a type we can reuse in other scripts using registry,

Reply:
I will skip this for now and create separate issue on it. The format of the target can be specified in two ways:
```
br:contosoregistry.azurecr.io/bicep/modules/core/storage:v1
br/ContosoRegistry:bicep/modules/core/storage:v1
```